deploy/kubernetes/tracee.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app.kubernetes.io/name: tracee
+    app.kubernetes.io/instance: tracee-v1
+    app.kubernetes.io/version: "v0.5.0"
+    app.kubernetes.io/component: tracee
+    app.kubernetes.io/part-of: tracee
+    app.kubernetes.io/managed-by: aquasecurity
```

Comment:
I don't think manged-by means that, it means which tool manages this. we can remove it